lib/tools.py

```diff
@@ -325,7 +325,7 @@ def sort_csv(filename):
     csv_reader_infile = csv.reader(input_file)
 
     # The first entry is the header line
-    infields = next(csv_reader_infile)
+    infields = ""
```

Comment:
So the use of `next()` isn't compatible with Python 3? If so, that's fine, but then we can still get rid of `infields = ""` right?